src/viam/robot/client.py

```diff
@@ -108,6 +108,17 @@ def with_api_key(cls, api_key: str, api_key_id: str, **kwargs) -> Self:
             """
             Create RobotClient.Options with an API key for credentials and default values for other arguments.
 
+            ::
+
+                # Replace "<API-KEY>" (including brackets) with your machine's API key
+                api_key = '<API-KEY>'
+                # Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID
+                api_key_id = '<API-KEY-ID>'
+
+                opts = RobotClient.Options.with_api_key(api_key, api_key_id)
+
+                robot = await RobotClient.at_address('ADDRESS FROM THE VIAM APP', opts)
```

Comment:
My only comment is that for almost all of these docs, you display user-definable variables as `<SOME-VAR-NAME>`, but for all the addresses you've just done `"ADDRESS"` or `"ADDRESS FROM THE VIAM APP"`. Should those also be in angle brackets (`<>`) for consistency?